sql/core/src/main/scala/org/apache/spark/sql/functions.scala

```diff
@@ -3269,14 +3269,6 @@ object functions {
    */
   def raise_error(c: Column): Column = Column.fn("raise_error", c)
 
-  /**
-   * Throws an exception with the provided error class and parameter map.
-   *
-   * @group misc_funcs
-   * @since 4.0.0
-   */
-  def raise_error(c: Column, e: Column): Column = Column.fn("raise_error", c, e)
```

Comment:
It seems this API added no long.

Reply:
Not sure that I understand the comment. Can you please elaborate?

Reply:
https://github.com/apache/spark/pull/42985.